cardigan/stories/components/DownloadLink/DownloadLink.stories.tsx

```diff
@@ -1,4 +1,4 @@
-import DownloadLink from '../../../../catalogue/webapp/components/DownloadLink/DownloadLink';
+import DownloadLink from '../../../../common/views/components/DownloadLink/DownloadLink';
```

Comment:
I know it wasn't used here before but we have an alias/shortcut for these if you want! `@weco/common/views/components/DownloadLink/DownloadLink`

Reply:
does this apply to the `.mdx` file too?

Reply:
Yeah, should work there too! It's honestly nitpicking, so up to you 👍

Reply:
changes made!